DigitalOffice/Services/FileService/Commands/AddNewFileCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FileService.Commands.Interfaces;
+using FileService.Database.Entities;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using FileService.Repositories.Interfaces;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace FileService.Commands
+{
+    public class AddNewFileCommand : IAddNewFileCommand
+    {
+        private readonly IFileRepository repository;
+        private readonly IValidator<FileRequest> validator;
+        private readonly IMapper<FileRequest, DbFile> mapper;
+
+        public AddNewFileCommand(
+            [FromServices] IFileRepository repository,
+            [FromServices] IValidator<FileRequest> validator,
+            [FromServices] IMapper<FileRequest, DbFile> mapper)
+        {
+            this.repository = repository;
+            this.validator = validator;
+            this.mapper = mapper;
+        }
+
+        public Guid Execute(FileRequest request)
+        {
+            validator.ValidateAndThrow(request);
+            DbFile newFile = mapper.Map(request);
```

Comment:
why not to use `var` here?

Reply:
why not to use DbFile here?
I don't think its clear what type mapper maps request to. So I clarified it.

Reply:
If we will have different mappers, then type of object will be obviously

Reply:
I renamed request model names so now it's easier to understand what mapper maps to.